bin/rule.ml

```diff
@@ -96,6 +96,10 @@ let options_of_syntax = function
   | Some Mdx.Cram -> [ "--syntax=cram" ]
   | None -> []
 
+let options_of_section = function
+  | Some s -> [ Fmt.str "--section %S" s ]
```

Comment:
There's a compatibility issue with `Fmt.str` which is only available since `fmt.0.8.7` and `fmt.0.8.7` is for OCaml `>= 4.05`.

For this next release we'd like not to break compatibility with older OCaml versions. Could you replace this `Fmt.str` use with something else please?

Reply:
Build for 4.02 is green. Waiting for all the builds and merging.